website/docs/development/godoc/generic-interfaces.md

```diff
@@ -0,0 +1,337 @@
+<!-- Code generated by gomarkdoc. DO NOT EDIT -->
+
+# interfaces
+
+```go
+import "github.com/k-orc/openstack-resource-controller/internal/controllers/generic/interfaces"
+```
+
+## Index
+
+- [type APIObjectAdapter](<#APIObjectAdapter>)
+- [type BaseResourceActuator](<#BaseResourceActuator>)
+- [type Controller](<#Controller>)
+- [type CreateResourceActuator](<#CreateResourceActuator>)
+- [type DeleteResourceActuator](<#DeleteResourceActuator>)
+- [type ORCApplyConfig](<#ORCApplyConfig>)
+- [type ORCStatusApplyConfig](<#ORCStatusApplyConfig>)
+- [type ReconcileResourceActuator](<#ReconcileResourceActuator>)
+- [type ResourceController](<#ResourceController>)
+- [type ResourceHelperFactory](<#ResourceHelperFactory>)
+- [type ResourceReconciler](<#ResourceReconciler>)
+- [type ResourceStatusWriter](<#ResourceStatusWriter>)
+
+
+<a name="APIObjectAdapter"></a>
+## type [APIObjectAdapter](<https://github.com/k-orc/openstack-resource-controller/blob/main/internal/controllers/generic/interfaces/adapter.go#L25-L41>)
```

Comment:
Didn't you say you wanted to drop these links?

Reply:
I did, but it doesn't look feasible to remove them. If you don't specify the `--repository.X` arguments to gomarkdoc it defaults them based on local configuration. The only way it outputs nothing is if they're not specified *and* the defaulting method doesn't work.

We need the behaviour to be the same locally as in CI. As we can't make it consistently omit them in both places, I think we're stuck with consistently including them in both places. I suspect this will get irritating, because any PR which changes these line numbers will result in a lint error. Lets see how awful it is. We'd have to fix it upstream, but the project doesn't look very active lately.